src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private static final String SEPARATOR = ",";

```

Comment:
I skipped the part with the constants and scrolled down the code and then like "separator? Why do we need System.lineSeparator() here?!" so, wouldn't it be great to change its name to COMA_SEPARATOR or just COMA to describe it's function properly?